Grammar and wording fixes for 1.20 release notes (#305)
replicatedhq/kots.io: content/release-notes/1.20.0.md

  - Check for updates button is always available.
  - Current deployment status is included.
* Added resource limits and requests for Admin Console pods.
* Admin Consoles will use `NODE_PORT` environment variable if set (e.g. from kURL embedded clusters), instead of the default 8800.
* Port forwarding no longer uses the `app.k8s.io/v1beta1` Application spec and has been aligned with the [documentation](/vendor/config/dashboard-buttons/#provide-a-kots-application-spec).
* [SupportBundle](https://troubleshoot.sh/docs/support-bundle/collecting/) and [Preflight](https://troubleshoot.sh/docs/preflight/introduction/) specs are no longer served on unauthenticated endpoints and instead are provided to [support-bundle and preflight binaries](https://troubleshoot.sh/docs/#installation) via [secrets](/kotsadm/troubleshooting/support-bundle/).
* The version check endpoint now correctly reports the current application version and adds information about KOTS Admin Console version, Kubernetes version, app status, etc.
{{</changes>}}

{{<fixes>}}

> * When an airgap bundle that does not match the installed license is uploaded, show an error message that explicitly says this instead of "failed to verify bundle signature".

Author: It's always been the case that you'll fail to install bundles from different apps than the license
It's just the error message was horrible: https://github.com/replicatedhq/kots/pull/1096

> * When an airgap bundle that does not match the installed license is uploaded, show an error message that explicitly says this instead of "failed to verify bundle signature".

Reviewer: ```suggestion
* Added an explicit error message when an uploaded airgap bundle does not match the installed license.
```

* Airgap bundle uploads via web UI no longer use a lot of memory.
* Fixed a bug that caused airgap automated (unattended, headless) installs to pull the application from the internet.
* Fixed a bug that could cause Admin Console instance to become unusable if application installation failed under certain circumstances.
* Fixed a bug that caused registry template functions to return no data.
* Fixed a bug that caused [HasLocalRegistry](reference/template-functions/config-context/#haslocalregistry) and [LocalRegistryHost](/reference/template-functions/config-context/#localregistryhost) to return incorrect information.

> * Fixed a bug that caused [GitOps](/kotsadm/gitops/single-app-workflows/) to fail to clone an empty repo on initial setup.

Reviewer: ```suggestion
* Fixed a bug that caused [GitOps](/kotsadm/gitops/single-app-workflows/) to fail on initial setup if the repo is empty.
```

Reviewer: Is it really when it's "empty" or "unitialized"?

Author: I have no idea
I know the grammar/wording was off

* Fixed a troubleshoot bug where the result of a [secret collector](https://troubleshoot.sh/docs/collect/secret/) would have a missing key.

> * Fixed a troubleshoot bug where a [run collector](https://troubleshoot.sh/docs/collect/run/) would hang if the Pod was in state ImagePullBackoff.

Reviewer: ```suggestion
* Fixed a troubleshoot bug where a [run collector](https://troubleshoot.sh/docs/collect/run/) would hang if the Pod was in the `ImagePullBackoff` state.
```
